simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListFragment.kt

```diff
@@ -129,6 +129,10 @@ class AccountListFragment : Fragment(R.layout.account_list) {
     this.viewModel.accountEvents
       .subscribe(this::onAccountEvent)
       .let { subscriptions.add(it) }
+
+    if (this.parameters.accountID != null) {
+      this.listener.post(AccountListEvent.AccountSelected(this.parameters.accountID!!, barcode = this.parameters.barcode, comingFromDeepLink = true))
```

Comment:
There is an assumption here that if the fragment was created with an account ID, it is being opened through a deep link, but there is nothing that enforces that. I would rather be explicit and add `comingFromDeepLink` to `AccountListFragmentParameters`, and set it to true when coming from a deep link.

Reply:
Good call. I've changed it to use `comingFromDeepLink`.